### PR TITLE
chore(playgroud): add a build playground step

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

This has been lacking for a long time, our playgrounds are always out of date.

### CODE REVIEW

File: scripts/buildClients.ts [hunk 1]
```diff
@@ -34,7 +40,8 @@ async function buildClient(language: Language, gens: Generator[]): Promise<void>
       await run(`sbt --batch -Dsbt.server.forcestart=true +compile`, { cwd, language });
       break;
     case 'swift':
-      await run(`swift build -Xswiftc -suppress-warnings`, { cwd, language });
+      // make this work in the playground
+      if (!playground) await run(`swift build -Xswiftc -suppress-warnings`, { cwd, language });
```
Comment (shortcuts) [inline scripts/buildClients.ts:44]: can we stick to this syntax pls

```suggestion
      if (!playground) {
        await run(`swift build -Xswiftc -suppress-warnings`, { cwd, language });
      }
```

**Review by shortcuts — COMMENTED** [review]
such a nice addition!!!

File: scripts/buildClients.ts (hunk 1 above)
Comment (millotp) [reply scripts/buildClients.ts:44]: sure !

**Review by shortcuts — DISMISSED** [review]
long time awaited feature!!

File: scripts/buildClients.ts (hunk 1 above)
Comment (millotp) [reply scripts/buildClients.ts]: I added the eslint rule for the scripts folder

**Review by shortcuts — DISMISSED** [review]
❤️

**Review by shortcuts — APPROVED** [review]
ggggg

**Pull request merged**